portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -48,33 +52,42 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     comments.jsonList.clear();
 
     for (Entity entity : results.asIterable()) {
+      String toDisplay = entity.getProperty("timestamp").toString() + ": ";
       String fromDatastore = (String) entity.getProperty("title");
-      if (!comments.jsonList.contains(fromDatastore)) {
-        comments.jsonList.add(fromDatastore);
-      }
+      Translate translate = TranslateOptions.getDefaultInstance().getService();
+      Translation translation =
+        translate.translate(fromDatastore,
+                            Translate.TranslateOption.targetLanguage(comments.languageCode),
+                            Translate.TranslateOption.format("text"));
+      toDisplay += translation.getTranslatedText();
+      comments.jsonList.add(toDisplay);
```

Comment:
Are you going to return all the comments in the datastore? Or only return comments.limit number of comments as the maximum?